private/buf/bufcli/bufcli.go

```diff
@@ -267,6 +268,46 @@ func BindVisibility(flagSet *pflag.FlagSet, addr *string, flagName string) {
 	)
 }
 
+// BindPageSize binds the page size flag.
+func BindPageSize(flagSet *pflag.FlagSet, addr *uint32, flagName string) {
+	flagSet.Uint32Var(
+		addr,
+		flagName,
+		10,
+		`The page size.`,
+	)
+}
```

Comment:
Other uses of page size aren't using these in bufcli. Lets use the existing structure like

https://github.com/bufbuild/buf/blob/563fa8df55be0556f4cb905f5bf3d400935fb319/private/buf/cmd/buf/command/beta/registry/plugin/pluginversion/pluginversionlist/pluginversionlist.go#L69-L73
to keep things consistent here, then separately refactor to push all of these into `bufcli` in a different PR if needed to not leave us in a half half state.